docs/docs/Deployment/deployment-docker.md

```diff
@@ -5,11 +5,26 @@ slug: /deployment-docker
 
 This guide demonstrates deploying Langflow with Docker and Docker Compose.
 
+Three options are available:
+
+* The [Quickstart](#quickstart-with-a-sqlite-database) option builds a Langflow image with Langflow's default SQLite database.
```

Comment:
```suggestion
* The [Quickstart](#quickstart-with-a-sqlite-database) option builds a Langflow image with Langflow's default SQLite database.Create a Dockerfile with Langflow's default SQLite database. The SQLite database is stored in the container's filesystem, which means all data is lost when the container is stopped since no persistent volume is configured.
```

Reply:
I suggest we don't include SQLite as it's unnecessary details.